### PR TITLE
[QuickFix] Apply CAST() to number_of_items and amount_raw in tofu_bnb_events

### PR DESCRIPTION
**Pull request by Hosuke**
Attempt to fix `nft.trades` compatibility on DuneSQL.
Currently there is an issue when querying `nft.trades` on DuneSQL.
[Demo query](https://dune.com/queries/1819764?d=11)
image
Also mentioned the issue here https://github.com/duneanalytics/spellbook/commit/b72d9e294dd7cdccc5330e1d417c319b10f8cb52#r94491148
cc. @belen-pruvost @0xRobin @augustog

### CODE REVIEW

**Comment by 0xRobin** [comment]
good catch @Hosuke :+1:

**Comment by belen-pruvost** [comment]
Thank you @Hosuke!  This might be redundant, but in the spirit of explicit casting, could you also add explicit casting to double to the following fields?
- platform_fee_amount_raw
- platform_fee_amount
- platform_fee_amount_usd
- platform_fee_percentage
- royalty_fee_percentage

**Comment by belen-pruvost** [comment]
@0xRobin I noticed that the schema for `tofu_bnb` was not created on Databricks after the original PR was merged, and instead it is referenced as `none`. Do you know if there is something else that needs to be fixed on that front?

**Comment by 0xRobin** [comment]
> I noticed that the schema for `tofu_bnb` was not created on Databricks after the original PR was merged

Hmm I don't have any visibility on the databricks side but this probably means it's missing from the project yml.
Let me squeeze that in here.

**Comment by Hosuke** [comment]
> could you also add explicit casting to double to the following fields?

Yes, I will do this now.

**Review by belen-pruvost — APPROVED** [review]

**Comment by belen-pruvost** [comment]
Let's wait on the CI checks before merging

**Comment by 0xRobin** [comment]
sorry, just included another tweak. :see_no_evil:

**Comment by belen-pruvost** [comment]
This should now be working as expected

**Pull request merged**
